packages/calcite-components/src/components/accordion-item/readme.md

```diff
@@ -17,9 +17,11 @@ A child component of `calcite-accordion`, where some behavior is inherited from
 
 ## Slots
 
-| Slot | Description                                                                   |
-| ---- | ----------------------------------------------------------------------------- |
-|      | A slot for adding custom content, including nested `calcite-accordion-item`s. |
```

Comment:
These are all generated by Stencil, so does the linter "fix" these after a stencil build? Or should we ignore the readme files inside of components?

Reply:
FWIW we do have some manually written sections above the autogenerated part of the component readmes. 
Yeah I'd prefer if we let the linter make some consistency tweaks after Stencil generates the documentation for now. But I'll keep an eye on the doc update CI and ignore the component readmes if there are linting errors.

When I have a little extra time I still plan on reworking our internal documentation setup. One of the ideas I have involves taking advantage of the component readmes more and moving utils to their own folders with their own readmes.

If we do that, it would be nice if the component readmes could be linted w/o the autogenerated part causing issues.

> Hopefully we don't have to use the disable/ignore comments too often.

Yeah we should only use ignore comments for auto-generated content like the TOCs and extreme edge cases. The only reason I added those TOCs was to replace the ones that were hand made and out of date. But we could also remove them completely if they aren't necessary.